stable/drone/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v1
 name: drone
 home: https://drone.io/
 icon: https://drone.io/apple-touch-icon.png
-version: 2.7.0
+version: 2.7.1
 appVersion: 1.6.5
```

Comment:
this should be 1.6.2

Reply:
drone server is 1.6.5
drone agent is 1.6.2

so it's hard to tell what the appVersion should be really :)